proto/grp.proto

```diff
@@ -17,4 +20,45 @@ message Group {
 
   // True if this group has linked a Github token.
   bool github_linked = 4;
+
+  // Whether users from the owned domain should automatically
+  // be added to the group when they sign up (defaults to false).
+  //
+  // Not yet implemented.
+  bool auto_populate_from_owned_domain = 5;
+}
+
+message CreateGroupRequest {
+  // Group initialized with required fields except for ID.
+  //
+  // The required fields are documented in the Group proto.
+  Group group = 1;
```

Comment:
instead of taking a "Group" object here, can we take the fields we would use to create the group? Like:

message CreateGroupRequest {
  string name = 1;
  string owned_doman = 2;
  ...
}

Reply:
Makes sense -- done